core/src/main/res/values/string_arrays.xml

```diff
@@ -25,4 +25,16 @@
         <item>audiotrack</item>
         <item>opensles</item>
     </string-array>
+    <string-array name="quality_entries">
```

Comment:
Same thing here: these item names cannot be localized, so they should refer to `strings.xml`